From 8f3ccb8ef69fd38890cc1015941343331726c5f8 Mon Sep 17 00:00:00 2001
From: Christopher Degawa <ccom@randomderp.com>
Date: Wed, 18 Aug 2021 11:02:41 -0500
Subject: [PATCH 1/5] pc: remove CMAKE_CXX_IMPLICIT_LINK_LIBRARIES

It should be sufficient to instead tell dependents to link using the
C++ compiler instead.

Signed-off-by: Christopher Degawa <ccom@randomderp.com>
---
 loader/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/loader/CMakeLists.txt b/loader/CMakeLists.txt
index 4cb11472f..e6c856392 100644
--- a/loader/CMakeLists.txt
+++ b/loader/CMakeLists.txt
@@ -438,6 +438,7 @@ install(EXPORT VulkanLoaderConfig DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/Vulk
 # https://gitlab.kitware.com/cmake/cmake/-/issues/22621
 find_package(PkgConfig)
 if (PKG_CONFIG_FOUND)
+    set(PRIVATE_LIBS "")
     if(WIN32)
         if(MINGW)
             set(VULKAN_LIB_SUFFIX "-1.dll")
-- 
2.41.0

